### PR TITLE
Add edges to analysis h5

### PR DESCRIPTION
**Pull request by roomrys**
### Description
[describe your changes here]

### Types of changes

- [ ] Bugfix
- [x] New feature
- [ ] Refactor / Code style update (no logical changes)
- [ ] Build / CI changes
- [ ] Documentation Update
- [ ] Other (explain)

### Does this address any currently open issues?
No

### Outside contributors checklist

- [x] Review the [guidelines for contributing](https://github.com/murthylab/sleap/wiki/Developer-Guide) to this repository
- [ ] Read and sign the [CLA](https://github.com/murthylab/sleap/blob/develop/sleap-cla.pdf) and add yourself to the [authors list](https://github.com/murthylab/sleap/blob/develop/AUTHORS)
- [x] Make sure you are making a pull request against the **develop** branch (not *main*). Also you should start *your branch* off *develop*
- [x] Add tests that prove your fix is effective or that your feature works
- [ ] Add necessary documentation (if appropriate)

#### Thank you for contributing to SLEAP!
:heart:

### CODE REVIEW

File: tests/info/test_h5.py [hunk 1]
```diff
@@ -17,6 +19,38 @@ def test_output_matrices(centered_pair_predictions):
...
+    assert node_names[12] == "midlegL1"
+    assert node_names[13] == "midlegL2"
+    assert node_names[14] == "midlegL3"
+    assert node_names[15] == "midlegR1"
+    assert node_names[16] == "midlegR2"
+    assert node_names[17] == "midlegR3"
+
+    for (src_node, dst_node) in edge_names:
+        assert src_node in node_names
+        assert dst_node in node_names
```
Comment (talmo) [inline tests/info/test_h5.py:52]: Check for correctness here instead.

```py
assert edge_names == centered_pair_predictions.skeleton.edge_names
```

if sequence comparison with lists doesn't work, try:

```py
assert tuple(edge_names) == tuple(centered_pair_predictions.skeleton.edge_names)
```

File: sleap/info/write_tracking_h5.py [hunk 2]
```diff
@@ -252,6 +260,7 @@ def main(labels: Labels, output_path: str, all_frames: bool = True):
     data_dict = dict(
         track_names=track_names,
         node_names=get_nodes_as_np_strings(labels),
+        edge_names=get_edges_as_np_strings(labels),
```
Comment (talmo) [inline sleap/info/write_tracking_h5.py:263]: Can we also save `skeleton.edge_inds` as a separate dataset (`/edge_inds`)? It's just an integer `(n_edges, 2)` array. It's redundant with `edge_names`, but it's a lot more convenient in some applications (e.g., plotting).

**Review by talmo — CHANGES_REQUESTED** [review]

File: tests/info/test_h5.py (hunk 1 above)
Comment (roomrys) [reply tests/info/test_h5.py:55]: Sequence comprehension works!

File: sleap/info/write_tracking_h5.py (hunk 2 above)
Comment (roomrys) [reply sleap/info/write_tracking_h5.py:263]: Yes, `edge_inds` have been added. Although, I wish I had more carefully looked at the skeleton attributes before using the `edge_names` attribute for the bento plugin. Anyway, having both `edge_names` and `edge_names` will be good for verification puposes.

**Review by talmo — APPROVED** [review]
LGTM!

**Pull request merged**